device/common/include/traccc/clusterization/device/tags.hpp

```diff
@@ -0,0 +1,13 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022-2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+namespace traccc::device {
+struct clustering_discard_disjoint_set {};
+struct clustering_keep_disjoint_set {};
```

Comment:
Please add some minimal documentation for the new types.

Reply:
Added.